circuit-lib/circuit-lib.js/src/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from "./merkle-tree";
-export * from "./elgamal-babyjubjub";
+// export * from "./elgamal-babyjubjub";
```

Comment:
should be uncommented again right?

Reply:
thx. fixed.  the fix for this included having to move precompute() in elgamal to its own file and not default export it, since its using a node only dependency (progress-bar). since its only used in the .test.ts file this is fine.

Reply:
we should put it in a separate sub package